ios/objective-c/AmpliObjectiveCSampleApp/AmpliObjectiveCSampleApp/Ampli/Ampli.m

```diff
@@ -1461,152 +663,152 @@ - (void)setGroup:(NSString *)name value:(NSString *)value extra:(MiddlewareExtra
 }
 
 - (void)setGroup:(NSString *)name value:(NSString *)value options:(EventOptions *_Nullable)options extra:(MiddlewareExtra *_Nullable)extra {
-    if (![self isInitializedAndEnabld]) {
+    if (![self isInitializedAndEnabled]) {
         return;
     }
     [self handleEventOptions:options];
     [self.client setGroup:name groupName:value];
 }
 
 - (void)flush {
-    if (![self isInitializedAndEnabld]) {
+    if (![self isInitializedAndEnabled]) {
         return;
     }
     if (self.client != nil) {
         [self.client uploadEvents];
     }
 }
-- (void)eventMaxIntForTest:(EventMaxIntForTestProperties *_Nonnull)properties options:(EventOptions *_Nullable)options extra:(MiddlewareExtra *_Nullable)extra {
+- (void)eventMaxIntForTest:(EventMaxIntForTest *_Nonnull)event options:(EventOptions *_Nullable)options extra:(MiddlewareExtra *_Nullable)extra {
     [self handleEventOptions:options];
-    [self track:[EventMaxIntForTest initWithEventProperties:properties] options:options extra: extra];
+    [self track:event options:options extra: extra];
 }
 
-- (void)eventMaxIntForTest:(EventMaxIntForTestProperties *_Nonnull)properties extra:(MiddlewareExtra *_Nullable)extra {
-    [self track:[EventMaxIntForTest initWithEventProperties:properties] extra: extra];
+- (void)eventMaxIntForTest:(EventMaxIntForTest *_Nonnull)event extra:(MiddlewareExtra *_Nullable)extra {
+    [self track:event extra: extra];
 }
 
-- (void)eventMaxIntForTest:(EventMaxIntForTestProperties *_Nonnull)properties {
-    [self track:[EventMaxIntForTest initWithEventProperties:properties] extra: nil];
+- (void)eventMaxIntForTest:(EventMaxIntForTest *_Nonnull)event {
+    [self track:event extra: nil];
 }
 
 - (void)eventNoProperties:(EventOptions *_Nullable)options extra:(MiddlewareExtra *_Nullable)extra {
     [self handleEventOptions:options];
-    [self track:[EventNoProperties initEvent] options:options extra: extra];
+    [self track:[EventNoProperties new] options:options extra: extra];
 }
 
 - (void)eventNoProperties:(MiddlewareExtra *_Nullable)extra {
-    [self track:[EventNoProperties initEvent] extra: extra];
+    [self track:[EventNoProperties new] extra: extra];
 }
 
 - (void)eventNoProperties {
-    [self track:[EventNoProperties initEvent] extra: nil];
+    [self track:[EventNoProperties new] extra: nil];
```

Comment:
@falconandy I had to switch to `new` instead of `init` for any events without constructor properties. Same for `EventWithConstProperties`.

![image](https://user-images.githubusercontent.com/5790872/149604342-1ec9e631-6b2a-4f74-b116-30286c6e953e.png)

Reply:
Fixed